web/includes/control_functions.php

```diff
@@ -20,9 +20,9 @@ function buildControlCommand( $monitor )
                 case 'focus' :
                 {
                     $factor = $_REQUEST['yge']/100;
-                    if ( $monitor['HasFocusSpeed'] )
```

Comment:
I knew this was an object thing and @connortechnology confirmed Monitor went from array to object.
https://github.com/ZoneMinder/ZoneMinder/pull/1040/files#diff-a63038416a07d6320efa13efef13405cR11